Templates/Kotlin/ResponseType.stencil

```diff
@@ -29,15 +29,13 @@ data class {{ name }}(
         {% endfor %}
     )
 
+    {% if fragmentSelections %}
     companion object {
-        const val typeName = "{{ graphQLName }}"
-
-        {% if fragmentSelections %}
         fun getSelections(operationVariables: Map<String, String>): List<Selection> {
```

Comment:
While in the area, I'd consider updating to a `selections` property accessor.

Reply:
dont think we can since we need to pass operationVariables

Reply:
You're totally right. I just ghosted over that when I saw get prefix. 👍